api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/VisitTranslatorImpl.java

```diff
@@ -59,8 +60,10 @@ public Encounter toFhirResource(@Nonnull Visit visit) {
 		if (visit.getLocation() != null) {
 			encounterLocationTranslator.toFhirResource(visit.getLocation());
 		}
+
 		encounter.setClass_(mapLocationToClass(visit.getLocation()));
 		
+		encounter.getMeta().addTag(FhirConstants.OPENMRS_FHIR_EXT_ENCOUNTER_TAG, "Visit", "encounter-type-display");
```

Comment:
The tag itself should probably be "visit" or "encounter"; the display property should be the capitalised version of them, e.g. "Visit" or "Encounter" (`display` is usually what gets displayed to the user).

Reply:
@ibacher Will change it.

Reply:
@ibacher will the code value remain the same?

Reply:
@Medhavi-16 I'm not quite sure I understand the question, but I think this line should read:

```java
encounter.getMeta().addTag(FhirConstants.OPENMRS_FHIR_EXT_ENCOUNTER_TAG, "visit", "Visit");
```